### PR TITLE
browserify blog post

### PR DESCRIPTION
**Pull request by nathanharper**

### CODE REVIEW

File: blog/_posts/2015-02-15-field-notes.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
+---
+title: Taming JavaScript
```
Comment (dirkkelly) [inline blog/_posts/2015-02-15-field-notes.md]: Browserify and javascript

File: blog/_posts/2015-02-15-field-notes.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+layout: post
+author: nathanharper
+---
+
+When I joined InterExchange back in October, [App] was on the brink of a radical change --
+[Dirk] had recently set up the separate [Styleguide] project to manage our assets,
```
Comment (dirkkelly) [inline blog/_posts/2015-02-15-field-notes.md]: needs a link, please refer to me as DirkKelly and link to github.com/dirkkelly

**Comment by nathanharper** [comment]
:bowtie:

**Comment by dirkkelly** [comment]
:+1:

**Pull request merged**